google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiSelectorPanel.java

```diff
@@ -67,12 +71,25 @@
   private JLabel modulesLabel;
   private ModulesComboBox modulesComboBox;
 
+  private final Map<CloudLibrary, CloudApiManagementSpec> apiManagementMap;
```

Comment:
Feels like immutable map after filling with initial state?

Reply:
done (the instance created in the constructor is now immutable)

Reply:
Maybe change interface to explicit `ImmutableMap` so it's easy to see/catch attempts to modify?

Reply:
unless you feel strongly about it, I think its ok: if I were to change it I'd probably have to update all the getters too, and also update the `libraries` list too for consistency. I'm not sure if we need to expose the implementation as part of the api.

Reply:
That's fine with me as well.